plugins/inputs/ceph/README.md

```diff
@@ -208,9 +232,21 @@ All admin measurements will have the following tags:
     - total_bytes (float)
     - total_used_bytes (float)
     - total_avail_bytes (float)
-    - total_space (float, exists only in ceph <0.84)
-    - total_used (float, exists only in ceph <0.84)
-    - total_avail (float, exists only in ceph <0.84)
```

Comment:
I agree this seems indeed a very old version, but we should not remove it until next major release.

Reply:
Re-added with latest changes.